charts/everest/templates/hooks.yaml

```diff
@@ -1,3 +1,27 @@
 {{- if .Values.upgrade.preflightChecks }}
 {{- include "everest.preUpgradeChecks" (dict "namespace" (include "everest.namespace" .) "version" .Chart.Version "versionMetadataURL" (include "everest.versionMetadataURL" .)) }}
 {{- end }}
+---
+{{- /*
+# --- Pod Scheduling Policies
+# During the installation of Everest v1.7.0 we need to create default pod scheduling policies.
+*/}}
+
+{{- if  .Release.IsInstall }}
```

Comment:
Why only during installation?

Reply:
I saw that, what I meant to ask that if it needs to be created during install and upgrade (basically always), why even have this condition?

Reply:
because in the next release the whole block 
```
{{- /* if and (.Release.IsUpgrade) (eq .Chart.Version "1.7.0") */}}
...
{{ - end }}
```

will be deleted as soon as it won't be needed any more